scripts/2.TCGA-process.py

```diff
@@ -59,11 +60,118 @@
 snp_mutation_df.effect.value_counts().reset_index()
 
 
+# ### Convert mutation gene symbol labels to Entrez IDs  
+
+# Goal: Relabel the mutation data frame with Entrez IDs instead of gene names, by mapping a combination of chromosome and gene symbol to Entrez ID. The NCBI file downloaded and read in the next cell contains the Entrez ID - gene symbol pairs we will use to do so.
+
+# In[7]:
+
+base_url = 'ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/'
+map_name = 'Homo_sapiens.gene_info.gz'
+map_url = base_url + map_name
+path = os.path.join('download', map_name)
+urlretrieve(map_url, path)
+
+path = os.path.join('download', 'Homo_sapiens.gene_info.gz')
```

Comment:
use `map_name` variable here instead of 'Homo_sapiens.gene_info.gz' ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself "Don't repeat yourself"))